docs/how-tos/configure.md

```diff
@@ -82,7 +87,7 @@ For example, if you have a DC motor, follow the [corresponding configuration ins
 
 <!-- markdownlint-disable MD036 -->
 
-**5. Test your components**
+**5. Control your components**
 
 When you configure a component, a remote control panel is generated for it in the **CONTROL** tab of the Viam app.
```

Comment:
I think with the test section in existence, it makes sense to mention it here. Something like, 
```suggestion
When you configured each component, you saw the **TEST** panel on its configuration panel.
You can also access the control interfaces for all your components in one place from the **CONTROL** tab.
```